docs/webui_tutorial.rst

```diff
@@ -0,0 +1,74 @@
+Web Application Tutorial
+############################
+
+.. toctree::
+  :maxdepth: 2
+
+In order to quickly give people an overview and idea of what the Raiden Developer Preview is capable of, a simple web application has been created. This application utilizes the :doc:`Raiden REST API endpoints <rest_api>` to allow the user to interact with tokens networks, transfer tokens, see the current status of open channels along with closing and settling channels to name some of the functionalities. For a more specific guide of the API itself see the :doc:`API Walkthrough <api_walkthrough>`.
+
+The main focus of the web application is to display functionality and not to look pretty. Should you however want to make it pretty, don't hesitate to create a `pull request <https://github.com/raiden-network/raiden/pulls>`_.
+
+
+Running the web application
+---------------------------
+There are a few simple steps that needs to be carried out in order to run the Raiden web application. Firstly, a Raiden node needs to be setup. See :doc:`overview_and_guide` for instructions on this.
+
+The only dependency needed to run the web application is `node.js <https://docs.npmjs.com/getting-started/installing-node>`_.
+
+Once a Raiden node is up and running and node.js has been installed, it is quite easy to start the web application::
+
+    cd raiden/ui/web
+
+Followed by::
+
+    npm install
+
+And then to run the application simply do::
```

Comment:
What do you mean by "before isntall Raiden" in the above? It shouldn't matter if you run `python setup.py compile_webui` before or after you run `python setup.py install`, right?